examples/docker-for-mac/my_test.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-readonly IMAGE_TAG=v2.3.3
+readonly IMAGE_TAG=v2.4.0
```

Comment:
this version is not available yet

Reply:
Reading this doc I was under the impression that the docs should be updated before the creation of the tag
https://github.com/helm/chart-testing#releasing

Should the tag be created first? Can I even create a tag?

Reply:
It's in preparation for this version. So, I this is fine.

Reply:
Tag creation is part of the release process. You can't do that.